crates/ra_syntax/src/grammar/expressions.rs

```diff
@@ -158,13 +168,13 @@ fn current_op(p: &Parser) -> (u8, Op) {
 // Parses expression with binding power of at least bp.
 fn expr_bp(p: &mut Parser, r: Restrictions, bp: u8) -> BlockLike {
     let mut lhs = match lhs(p, r) {
-        Some(lhs) => {
+        Some((lhs, macro_blocklike)) => {
```

Comment:
Let's rename to  `blocklike`, it's not only about macros

Reply:
Sure, just missed that one